### DISCUSSION TITLE
"Paint on Template" gets blurry

### DISCUSSION THREAD

**Discussion by Hofkart** [OpenOrienteering/mapper#2165] [Questions & Answers]
Hi,
The "Paint on Template"  seems convinient in field work, with a Android device.
I have learnd, that if "pen" does not draw, you are outside the template, and have to create an additional.
When looking on a template file, in another tool, they are mostly black. I may depend of that the background is transparent. Not verified.

But my problem: When draving, lines are clear and thin. But (when saved?), lines becomes blurry and wide, and almost not readable. What to do to fix this?

**Comment by mlerjen** [comment]
**A simple workflow to use "Paint on template" is to:**
1) On desktop: Save the entire map area as .png, therby choose a resolution that suits you. The higher the resolution the finer the "field sketch" will be. My default is 256 dpi in my setup.
2) Add the .png as template 
3) Switch to "paint on template" mode and erase the content of the .png
3b) I usually draw a frame around along the templates border
![image](https://github.com/OpenOrienteering/mapper/assets/15611653/d6257c58-2664-480a-8f39-718bcc9e445f)


Now you got your mylar ready.

4) Transfer to the Android device

-> the coarseness of you field note is the result of the sketch templates resolution
-> some software replaces transparent with black. e.g. OCAD does.

**Reply by Hofkart** [reply]
Aha. You create your own georeferenced sketch area, with your own resolution and size. And get rid of the default size and resolution. I did not think of that all backgrounds are generalised and treated in the same way.

Anyhow, default is 1,5 km and 1000px, or 100mm at 1:15000. More or less the same as 256dpi. Then I think 600 dpi will be good for me.
Readability differs between different software. Open in Win Paint and save as tif, is one of the better, if you will use the sketch in OCAD.
My favorite, IrfanView, failed this time.

Thank you very much for the lesson.

**Reply by mlerjen** [reply]
You also can export the Sketch from Mapper directly 
1) Open the Template dialog, hide everything (incl. the map) but the sketch layer
2) File>Export as> Image 
3) check  Show templates, check Save world file,  select resolution, press export and chose the file format from the save as dialog.

(no need to mention that you miss out the most fun part by drawing in OCAD :-) but I know, the power of habit...)

**Comment by dg0yt** [comment]
The templates are raster graphics, so they are saved with a "reasonable" pixel size which is much less than your modern zoomed-in screen resolution. Otherwise the raster graphics would put a heavy load on memory. You may face the operating system closing the app unexpectedly.

You can always use vector graphics by defining and using dedicated map symbols, maybe used in a separate "map part".
